qcschema/dev/wavefunction/scf_wavefunction.py

```diff
@@ -119,3 +151,67 @@
     "shape": ["nmo"]
 }
 
+
+# Electron-Repulsion Integrals
+scf_wavefunction["scf_eri_aa"] = {
```

Comment:
Can you elaborate on what you mean by an SCF associated, spin dependent ERI tensor in the AO basis? For non-relativistic calculations, ERIs are spin-independent, and AO quantities are typically not tied to an SCF solution. I just want to make sure that if we fill this out, we're populating things in the proper way.

Reply:
My bad, I think I was a tad too ambitious when copying the MO dataset and simply replacing `MO` with `AO`..
I will update this tomorrow to only include a single `eri` field, just like you suggest :+1: